homeassistant/components/withings/common.py

```diff
@@ -1030,6 +1030,21 @@ async def async_get_data_manager(
     if const.DATA_MANAGER not in config_entry_data:
         profile = config_entry.data.get(const.PROFILE)
 
+        if CONF_CLOUDHOOK_URL not in config_entry.data:
```

Comment:
We have to check if there is an active subscription, otherwise taking this path has no use